Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "ISC"
 name = "songbird"
 readme = "README.md"
 repository = "https://github.com/serenity-rs/songbird.git"
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
Please don't change this: we'll handle this separately when we make a release at some later time.